rules/linux/execution_linux_netcat_network_connection.toml

```diff
@@ -24,6 +24,59 @@ index = ["auditbeat-*", "logs-endpoint.events.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Netcat Network Activity"
+note = """## Triage and analysis
+
+### Investigating Netcat Network Activity
+
+Netcat is a dual-use command line tool that can be used for various purposes, such as port scanning, file transfers, and
+connection tests. Attackers can abuse it's functionality for malicious purposes such create bind or reverse shells to
```

Comment:
This revision is slightly repetitive, but I think it's clearer — I stumbled several times over "create bind or reverse shells" before I understood it:
```suggestion
connection tests. Attackers can abuse its functionality for malicious purposes such creating bind shells or reverse shells to
```